pombola/za_hansard/migrations/0007_create_terms_for_questions.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+from pombola.south_africa.models import ParliamentaryTerm
+from pombola.za_hansard.models import Question
+
+
+
+def forwards_func(apps, schema_editor):
```

Comment:
I was thinking we should first delete all of the data from the questions and answers tables because some of the data is incorrect since we've been identifying duplicates incorrectly. So this migration won't be needed since the will be nothing in the table, but I added it just in case.

Reply:
sjoe I'm nervous of deleting all questions in case the code can't import them as effectively as before because of changes over time. Can we rather just delete those with duplicates?

Reply:
Good idea :+1: I'll write a script for that.